### ISSUE TITLE
Eval metrics in DeepLab Tensorboard

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#647]
It's not clear if the total loss in Tensorboard when using Deeplab is a validation loss or training loss (or if the images in the images tab are from the training or validation set). Also, I was expecting there to be some explicit validation metrics in Tensorboard. (For object detection, there's a validation mAP that shows up). The task is to see what's going on and possibly log this information if possible.

**Comment by lewfish** [comment]
The problem motivating this card is that it's not clear when to stop training / how many steps to use.
